CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,30 @@
 
 import Foundation
 
-print("Hello, World!")
+func print_icecream (width: Int, height: Int) {
```

Comment:
함수명을 잘 지어주셨군요! 하지만, API Design Guidelines의 [General Conventions](https://www.swift.org/documentation/api-design-guidelines/#general-conventions)파트에서 보면, 스위프트의 함수명은lowerCamelCase를 따르도록 지시하고 있습니다! 문서를 읽어보시고 그에 맞게 수정해보세요!

더불어, 함수 다음 매개변수를 작성하기 위한 괄호는 띄어쓰지 않고 붙여주어야 합니다!
```suggestion
func print_icecream(width: Int, height: Int) {
```

Reply:
감사합니다!! 수정하겠습니다!👍